script/Params.s.sol

```diff
@@ -26,20 +26,18 @@ import {WAD, RAY, RAD} from '@libraries/Math.sol';
 // --- Utils ---
 
 // to replace OP voting token
-bytes32 constant FTRG = bytes32('FTRG'); // 0x4f50000000000000000000000000000000000000000000000000000000000000
+bytes32 constant ARB = bytes32('ARB'); // 0x4f50000000000000000000000000000000000000000000000000000000000000
```

Comment:
- [ ] Please provide the bytes32 string for each collateral for me to update the sdk
- [ ] Please update the comments with the correct values here